lib/LANraragi/Plugin/Metadata/EHentai.pm

```diff
@@ -139,7 +141,7 @@ sub lookup_gallery {
         my ( $gId, $gToken ) = &ehentai_parse( $URL, $ua );
 
         if ( $gId ne "" && $gToken ne "" ) {
-            return ( $gId, $gToken );
+            die "Reverse Image Search failed\n";
```

Comment:
This is incorrect, the sub is meant to return early if it finds a gID/gToken.  
Any errors would just bubble up from `ehentai_parse` now I believe

Reply:
Yes, I misread the negation... :man_facepalming: